sql/src/main/java/io/crate/execution/ddl/tables/AlterTableOperation.java

```diff
@@ -494,7 +496,12 @@ private static DiscoveryNode getNodeForResize(ClusterState state) {
 
     private CompletableFuture<Long> renameIndicesBulk(final List<RenameIndexAction> renameIndexActions) {
         BulkRenameIndexRequest request = new BulkRenameIndexRequest(renameIndexActions);
-        FutureActionListener<AcknowledgedResponse, Long> listener = new FutureActionListener<>(r -> 0L);
+        FutureActionListener<AcknowledgedResponse, Long> listener = new FutureActionListener<>(response -> {
+            if (!response.isAcknowledged()) {
+                throw new RuntimeException("Rename Index request was not acknowledged");
```

Comment:
Please include some more details what this means and what the user can do about it.